### PR TITLE
COR-1624-remove-KPI-from-GPs

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

* 
* Remove the KPI tiles from the "General practitioner" page
* Sanity changes are done in [this PR](https://github.com/minvws/nl-covid19-data-dashboard/pull/4775)
* 

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-05-26 at 11 51 01

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-05-26 at 11 50 44

</details>

### CODE REVIEW

**Comment by VWSCoronaDashboard18** [comment]
@VWSCoronaDashboard28, like it is said in the PR summary the sanity changes are combined in another PR:

https://github.com/minvws/nl-covid19-data-dashboard/pull/4775

File: packages/app/schema/archived_nl/__difference.json
Comment (VWSCoronaDashboard26) [inline packages/app/schema/archived_nl/__difference.json:1]: While I understand that it makes little sense to keep this file around when we are removing the `doctor___xxx` properties (which are the only properties in here), I think removing it basically omits some of the work @VWSCoronaDashboard25 has put into [COR-1588](https://github.com/minvws/nl-covid19-data-dashboard/pull/4762). This file is the part of the new way of archiving data, hence I think it is a nice pointer to keep around for when data is being archived again.

Feel free to resolve if you do not agree.

Comment (VWSCoronaDashboard25) [reply packages/app/schema/archived_nl/__difference.json:1]: True, but it's okay. Leaving this file open but empty would be weird too. Besides that, it's just a small copy+paste exercise so no biggy.

Comment (VWSCoronaDashboard18) [reply packages/app/schema/archived_nl/__difference.json:1]: Understandable, I had this thought as well. But I discussed this with #25 before and like he said this was fine to be picked up.

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Review by VWSCoronaDashboard27 — APPROVED** [review]

**Pull request merged**